addons/binding/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/gson/KonnectedModulePayload.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.konnected.internal.gson;
+
+import java.util.HashSet;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link KonnectedModulePayload} is responsible to hold
+ * data to send as settings for the konnected module
+ *
+ * @author Zachary Christiansen - Initial contribution
+ *
+ */
+public class KonnectedModulePayload {
+
+    public KonnectedModulePayload(String authTokenPassed, String apiURLPassed) {
+        this.authToken = authTokenPassed;
+        this.apiUrl = apiURLPassed;
+        this.sensors = new HashSet<KonnectedModuleGson>();
+        this.actuators = new HashSet<KonnectedModuleGson>();
+    }
+
+    @SerializedName("sensors")
+    HashSet<KonnectedModuleGson> sensors;
```

Comment:
Use the interface here instead of the type so this and other occurrences should be a `Set`